packages/plugin-desk-exchange/README.md

```
@@ -0,0 +1,97 @@
+# DESK Exchange Plugin for Eliza
+
+This plugin enables interaction with the DESK Perpetial DEX through Eliza, providing perpetual futures trading capabilities. Visit http://desk.exchange/ for more details.
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)